entities/weapons/raft_builder/cl_init.lua

```diff
@@ -2,10 +2,25 @@ include( "shared.lua" )
 
 local abs = math.abs
 
-local GHOST_COLOR = Color( 0, 255, 0, 150 )
+local GHOST_VALID = Color( 0, 255, 0, 150 )
+local GHOST_INVALID = Color( 255, 0, 0, 150 )
 local GHOST_INVIS = Color( 0, 0, 0, 0 )
 local INPUT_DELAY = 0.2
 
+surface.CreateFont( "RVR_RaftBuilderIngredients", {
+    font = "Bungee Regular",
+    size = ScrH() * 0.045,
+    weight = 500
+} )
+
+local raftPartIcons = {
+    raft_foundation = Material( "rvr/icons/raft_foundation.png" ),
+    raft_platform = Material( "rvr/icons/raft_platform.png" ),
+    raft_stairs = Material( "rvr/icons/raft_stairs.png" ),
+    raft_wall = Material( "rvr/icons/raft_wall.png" ),
+    raft_fence = Material( "rvr/icons/raft_fence.png" )
+}
```

Comment:
In terms of making the game "moddable", might it be nice to allow for new parts to be added?